dbms/src/Storages/BackgroundProcessingPool.cpp

```diff
@@ -155,6 +155,7 @@ void BackgroundProcessingPool::threadFunction(size_t thread_idx)
     }
 
     auto memory_tracker = MemoryTracker::create();
+    memory_tracker->setNext(root_of_mem_trackers_not_managed_by_process_list.get());
```

Comment:
Looks like `root_of_mem_trackers_not_managed_by_process_list` is only used by `BackgroundProcessingPool`, why not just define it as a static member in `BackgroundProcessingPool`?

Reply:
for further usage，we need a “other” mem tracker to control those not managed by ProcList

Reply:
OK, maybe we can rename it to `root_of_non_query_mem_trackers`?

Reply:
good idea

Reply:
done